feature/experimental/gribi/gribi.go

```diff
@@ -0,0 +1,209 @@
+package gribi
```

Comment:
done

Reply:
How is this all different from
https://github.com/openconfig/gribigo/blob/main/fluent/fluent.go
?

I'm not sure we should have two fluent gribi APIs in openconfig.

Reply:
I think the value proposition of this package is that it streamlines modify-await-chk that you would otherwise have to do separately using the fluent client. The modify and chk have to agree on the entries being programmed, and this package provides a way to batch program these entries. This is a more friendly way to write gRIBI tests.